gcs_to_bq/common/gcs_to_bq_util.py

```diff
@@ -14,6 +16,9 @@ def append_dataframe_to_bq(frame, column_types, dataset, table_name):
                    names must match the columns in the DataFrame
      dataset: The BigQuery dataset to write to
      table_name: The BigQuery table to write to"""
+  if col_modes is None:
+    col_modes = {}
+
```

Comment:
Could we just set `col_modes={}` in the function signature?

Reply:
That's one of the annoying gotchas in python - the default arg is reused across multiple invocations, so it's recommended to set to None for mutable objects. See https://docs.python-guide.org/writing/gotchas/